workflow.txt

```diff
@@ -1,10 +1,6 @@
-git clone
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
+(make changes)
 git add .
-git pull upstream master
-git merge my-solution
-fork on github
+git commit -m "(write message)"
 git checkout master
-git push origin my-solution
+git merge my-solution
```

Comment:
Gotcha. I had the sense while I was answering this one that it was open to interpretation, and that I might very well have been misinterpreting it. I'm definitely used to this now, since it's our homework submission workflow.